cmd/kubeadm/app/preflight/checks.go

```diff
@@ -98,7 +98,12 @@ func (CRICheck) Name() string {
 
 // Check validates the container runtime through the CRI.
 func (criCheck CRICheck) Check() (warnings, errors []error) {
-	if err := criCheck.exec.Command("sh", "-c", fmt.Sprintf("crictl -r %s info", criCheck.socket)).Run(); err != nil {
+	crictlPath, err := criCheck.exec.LookPath("crictl")
```

Comment:
how do you feel about putting this in a single function?

Reply:
That would be tedious IMO. LOCs will not get changed. WDYT? @jamiehannaford